### PR TITLE
(#7 and #9) - Add ci.yml workflow

### PR DESCRIPTION
**Pull request by mfcorgi**
- Consolidated dotnet-core.yml and release-notes.yml into ci.yml. (PR for #7 and #9)

This worfklow will work on pull_requests and pushes to main and releases branches. 

- It Will update the release draft only on push.
- It will always upload the alpha artifacts to the workflow run.

The idea is to create another workflow to transform the draft release into an actual itself. It could be when pushing a release branch into master.

### CODE REVIEW

File: .github/workflows/dotnet-core.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    
+      - name: Display updated csproj File
+        run: cat Freshli.Cli/Freshli.Cli.csproj
+
+      - name: Build
+        run: dotnet build --configuration Release
+                
+      - name: Package
+        if: github.event_name == 'push'
+        run: dotnet pack -c Release
```
Comment (mrbiggred) [inline .github/workflows/dotnet-core.yml:51]: The CLI does not produce a NuGet package.  The "Package" and "Publish Alpha Package to GitHub Packages" can be removed.

**Review by mrbiggred — CHANGES_REQUESTED** [review]
The Command Line Interface (CLI) does not produce a NuGet package.  The steps that pack and publish the NuGet package should be removed from the build.

The result of this build should be an executable (exe) with the correct version number set.  Case #9 is to put the built exe somewhere after it is built.

**Review by mrbiggred — CHANGES_REQUESTED** [review]
This looks the same as PR #17.  Should one of the PRs be discarded?

**Comment by mfcorgi** [comment]
Closing this PR because the ci.yml file was already pushed in #17

**Pull request closed without merging**